tests/test_debug_hex_format.rs

```diff
@@ -0,0 +1,102 @@
+use skipgraph::core::{Identifier, MembershipVector, Address};
+use skipgraph::core::model::identity::Identity;
+
+#[test]
+fn test_identifier_debug_shows_hex_not_raw_bytes() {
+
+    let id_bytes = [69, 11, 103, 102, 141, 75, 166, 128, 3, 116, 40, 7, 102, 211, 4, 44, 26, 234, 34, 150, 21, 99, 236, 216, 142, 116, 70, 33, 143, 133, 174, 83];
+    let identifier = Identifier::from_bytes(&id_bytes).unwrap();
+    
+    let debug_output = format!("{:?}", identifier);
+    let expected_hex = hex::encode(id_bytes);
+    
+    // Should show hex format, not raw byte array
```

Comment:
Should we also add one line of?
```
assert_eq!(identifier, expected_hex);
```

As literally we expect `identifier == expected_hex == debug_output`

Reply:
Maybe I am missing something here, but we already have a check `assert_eq!(debug_output, expected_hex);` which is equivalent to your suggestion, since `debug_output` is indeed `identifier` in the debug format. Do we need to add something extra?